### PR TITLE
Bring back dymo-label, dymo-stamps

### PR DESCRIPTION
**Pull request by claui**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[closed issues]: https://github.com/Homebrew/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

___

Both casks got [deleted](https://github.com/Homebrew/homebrew-cask/pull/36884) in 2017 due to `dymo-label`’s installation failing; cf. #36883. Given that the issue has been fixed in https://github.com/Homebrew/brew/pull/4265, **this PR re-adds both casks.**

Also, this PR:

- fixes the homepage for `dymo-stamps`, attributing the product to [Endicia, its actual vendor](https://www.endicia.com/landing-pages/dymo-software/dymostamps);

- updates `dymo-label` to the current version, 8.7.2; and

- in `dymo-label`, adds a missing entry `com.dymo.dls.webservice` to the launchctl uninstall list.

### CODE REVIEW

**Comment by commitay** [comment]
This should probably be in [`Homebrew/cask-drivers`](https://github.com/Homebrew/homebrew-cask-drivers) now.

**Comment by claui** [comment]
@commitay Good point. I have no strong opinion on that.

The upstream package for `dymo-label` contains both a set of drivers and an app. The app presents itself as a (rather primitive) design tool but is tightly coupled with the family of DYMO printers. It refuses to launch unless one of their printers is connected.

I’d be fine with relocating the cask, along with its dependent, `dymo-stamps`.

@Homebrew/cask OK with me re-submitting this to [Homebrew/homebrew-cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers)?

**Comment by commitay** [comment]
Just adding a note that this shouldn't be merged here or in drivers until the next brew tag as it's only fixed on master.

**Comment by claui** [comment]
@commitay Thanks for the heads up! I wasn’t aware that `brew update` won’t fast-forward to master by default. 😆

File: Casks/dymo-label.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  name 'Dymo Label'
+  homepage 'https://www.dymo.com/en-US/online-support'
+
+  pkg "DYMO Label v.#{version.major}.pkg"
+
+  uninstall launchctl: [
+                         'com.dymo.dls.webservice',
+                         'com.dymo.pnpd',
+                       ],
+            pkgutil:   [
```
Comment (commitay) [inline Casks/dymo-label.rb]:
```
            pkgutil:   [
                         'com.dymo.cups',
                         'com.dymo.dls.*',
                       ]
```

File: Casks/dymo-label.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                         'com.dymo.dls.application',
+                         'com.dymo.dls.appsupport',
+                         'com.dymo.dls.documents',
+                         'com.dymo.dls.frameworks',
+                         'com.dymo.dls.npapi.addin',
+                         'com.dymo.dls.office.addins',
+                         'com.dymo.dls.safari.addin',
+                       ]
+
+  zap delete: [
```
Comment (commitay) [inline Casks/dymo-label.rb]: `zap trash`

**Comment by claui** [comment]
@Homebrew/cask Which would be the correct repository? homebrew-cask or homebrew-cask-drivers?

**Comment by commitay** [comment]
https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md

> Driver: Software to make a hardware peripheral recognisable and usable by the system. If the software is useless without the peripheral, it’s considered a driver.

As it doesn't work without the printer these should be in `drivers`. Can you submit them there as separate PRs so they can be tested by CI please?

File: Casks/dymo-stamps.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'dymo-stamps' do
+  version :latest
```
Comment (commitay) [inline Casks/dymo-stamps.rb]: This can be versioned with an appcast.

https://s3.amazonaws.com/endiciaformac/EndiciaForMacSparkle.xml

**Comment by zbeekman** [comment]
@claui & @vitorgalvao : I'm interested in this too. It looks like audit is flagging this due to homepage and download TLD being different. Also, these should be migrated to drivers, no? I can help update this.

**Comment by claui** [comment]
@zbeekman Thanks for your thoughts! No idea why I let this slip. Yes, I’m still super interested in bringing those casks back!

Closing here. Feel free to re-submit on drivers whenever you get around to it!

**Comment by zbeekman** [comment]
@claui I've never used dymo-stamps; does it drive the same label/maker printer device? Or a different device? (i.e. do they both belong in cask-drivers?)

Also, I don't follow commitay's comment about app-cast. Maybe I need to read the cask documentation a bit.

**Comment by claui** [comment]
> I've never used dymo-stamps; does it drive the same label/maker printer device?

@zbeekman Haven’t used it either. The only thing I know is that it’s a third-party product. I don’t see a strong case to revive `dymo-stamps`if you ask me.

About the version/appcast comment: that was to let me know that the vendor’s website offers versioned URLs. (We prefer those because they’re checksum-friendly.) My commit has already been updated to reflect that.

**Comment by alexg0** [comment]
> > I've never used dymo-stamps; does it drive the same label/maker printer device?
> @zbeekman Haven’t used it either. The only thing I know is that it’s a third-party product. I don’t see a strong case to revive `dymo-stamps`if you ask me.
> 

@claui & @zbeekman As a user of both of these products, putting a few words in support of `dymo-label` and `dymo-stamps`.  They drive a series of label printers from Dymo, some models are compatible with one of the two or both.  

Would hope that both are brought back since arguments for both are same, and I use both.

**Comment by claui** [comment]
@alexg0 Thanks, good to know there’s at least some demand for both casks!

**Comment by zbeekman** [comment]
1/2 (DYMO-Label) in https://github.com/Homebrew/homebrew-cask-drivers/pull/728

**Comment by zbeekman** [comment]
@claui @Homebrew/cask Should dymo-stamps be renamed to endicia? It seems that the software is not really called dymo-stampsm anymore, although the nomenclature is confusing.

**Comment by claui** [comment]
@zbeekman Good catch! Yes, the cask should be named `endicia`.

Besides, it should be submitted to homebrew-cask rather than homebrew-cask-drivers because it [appears to support any printer](http://mac.endicia.com/extras/printers/) now, not just DYMO’s printers. That also means the dependency on `dymo-label` would need to go.

**Comment by zbeekman** [comment]
Yup. Figured as much. Thanks!

**Comment by claui** [comment]
@zbeekman Disregard what I said; the cask already exists on homebrew-cask. So I feel we’re done here. Thanks again for your help!

**Comment by zbeekman** [comment]
Yeah, it exists but is outdated. Also if there was a dymo-stamps do we need
to mark anything as a migration?

**Comment by claui** [comment]
@zbeekman The package’s been gone for such a long time, I don’t feel we need any migration hint.

**Pull request closed without merging**